packages/solid/web/server/index.ts

```diff
@@ -12,12 +12,15 @@ export {
   Match,
   Index,
   ErrorBoundary,
+  // This overrides mergeProps from dom-expressions/src/server.js
   mergeProps
 } from "solid-js";
 
 export const isServer: boolean = true;
 export const isDev: boolean = false;
 
+// Types for these come from dom-expressions/src/server.d.ts
+// These override the functions from dom-expressions that throw on the serverside.
 export function render() {}
 export function hydrate() {}
 export function insert() {}
```

Comment:
These functions override the error-throwing functions that are now exported from dom-expressions, making this particular set of functions be no-ops instead of throwing errors.

All the other newly-added functions in dom-expressions continue to throw errors if they are not overriden here, which is non-breaking. 

Removing the set of functions here could be breaking if anyone is already depending on them being no-ops, but the new functions added to dom-expressions/src/server.js never existed there, so the introduction of those new error-throwing functions cannot be breaking because no one used them before.

If desired, those additional functions in dom-expressions can be overriden here to make them no-ops instead of error-throwing.

Reply:
This seems fine to me and honestly I think I'm ok even making them throw if called. I don't see any reason why these should ever be called on the server with a server build.